projects/canopy/src/lib/forms/sort-code/sort-code.component.html

```diff
@@ -0,0 +1,73 @@
+<fieldset [attr.aria-describedby]="ariaDescribedBy">
+  <legend lg-label id="sortCodeLabel">
+    <ng-content></ng-content>
+  </legend>
+
+  <ng-content select="lg-hint"></ng-content>
+  <div
+    class="lg-sort-code__fields"
+    [formGroup]="sortCodeFormGroup"
+    #sortCodeFormDirective="ngForm"
+  >
+    <div class="lg-sort-code__input">
+      <label lg-label class="lg-visually-hidden" [for]="firstId"
+        >First two digits of your sort code</label
+      >
+
+      <lg-input-field lgMarginBottom="none">
```

Comment:
I was wondering whether there is too much repetition going on here and whether we could loop over a set of fields...I think I'm probably leaning towards this implementation though out of fear of it becoming over engineered. be good to hear what others think though...

Reply:
I'm more included to keep it separately implemented fields, rather than a loop, but happy to change it if there's a strong consensus to.

Reply:
I think I'm ok with this for now, just in terms of simplicity